aws/resource_aws_api_gateway_stage.go

```diff
@@ -384,3 +432,17 @@ func resourceAwsApiGatewayStageDelete(d *schema.ResourceData, meta interface{})
 
 	return nil
 }
+
+func flattenApiGatewayStageAccessLogSettings(accessLogSettings *apigateway.AccessLogSettings) []map[string]interface{} {
+	if accessLogSettings == nil {
+		return nil
+	}
+	item := make(map[string]interface{})
+	if accessLogSettings.DestinationArn != nil {
```

Comment:
Something to consider when working in the AWS provider is that the SDK provides helper functions so you don't need these wrapping `nil` checks (in most cases anyways): `item["destination_arn"] = aws.StringValue(accessLogSettings.DestinationArn)`